src/codex_plus/llm_execution_middleware.py

```
@@ -363,61 +430,356 @@ async def process_request(self, request, path: str):
+                        self._async_httpx_client = httpx.AsyncClient(timeout=30.0)
+
+                    async_client: httpx.AsyncClient = self._async_httpx_client
+                    response = await cleanup_stack.enter_async_context(
+                        async_client.stream(
+                            request.method,
+                            target_url,
+                            headers=minimal_headers,
+                            content=body if body else None,
+                        )
+                    )
+                except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)